scratchpad/README.md

```diff
@@ -0,0 +1,24 @@
+# Scratchpad
+
+This sample demonstrates a minimalistic single-file style that might be useful for quickly sketching out an idea, or for sharing a minimal reproduction of a bug.
+Production code should not be arranged like this! See the other samples in this repo for recommended project structure.
+
+### Running this sample
+
+In one terminal:
+
+```
+temporal server start-dev
+```
+
+In another terminal:
+
+```
+ts-node scratchpad.ts
```

Comment:
```suggestion
ts-node scratchpad.ts -w my-workflow-id -t my-task-queue
```

Reply:
Thanks. The CLI arg parsing was an experiment. I've decided to revert it, seeing as the point here is minimalism. So the workflow ID and task queue are top-level constants (and the workflow uses `WORKFLOW_ID_REUSE_POLICY_TERMINATE_IF_RUNNING`).